### PR TITLE
Improve default replacement values

### PR DESCRIPTION
**Pull request by ylboerner**
This PR improves the default replacement values that are set in the appsettings. Date fields for example now get assigned an actual date instead of a string.

### CODE REVIEW

File: Healex.HL7v2Anonymizer/appsettings.json [hunk 1]
```diff
@@ -70,75 +70,75 @@
                     },
                     {
                         "Path": "PID.11.5",
-                        "Value": "Zip code"
+                        "Value": "A104"
                     },
                     {
                         "Path": "PID.11.6",
                         "Value": "Country"
```
Comment (dd52) [inline Healex.HL7v2Anonymizer/appsettings.json:77]: v2 specification specifies an ID of max length 3 letter code here 'DEU' or so (https://hl7-definition.caristix.com/v2/HL7v2.5/Fields/PID.11.6)

**Review by dd52 — CHANGES_REQUESTED** [review]
great!

File: Healex.HL7v2Anonymizer/appsettings.json (hunk 1 above)
Comment (ylboerner) [reply Healex.HL7v2Anonymizer/appsettings.json]: Awesome, thank you!

**Review by dd52 — APPROVED** [review]

File: Healex.HL7v2Anonymizer/appsettings.json (hunk 1 above)
Comment (msdoege) [reply Healex.HL7v2Anonymizer/appsettings.json]: Is this about PID-11.5 zip code or the PID-11.6 country? I think you changed the wrong value there and "Country" should be changed.

**Pull request merged**